test/QuadForm/Herm/Lattices.jl

```diff
@@ -0,0 +1,94 @@
+@testset "Lattices" begin
+  
+  #
+  # A maximal integral lattice
+  #
+
+  Qx, x = PolynomialRing(FlintQQ, "x")
+  f = x^2 - 3
+  K, a = NumberField(f, "a", cached = false)
+  Kt, t = PolynomialRing(K, "t")
+  g = t^2 + 1
+  E, b = NumberField(g, "b", cached = false)
+  D = matrix(E, 3, 3, [1, 0, 0, 0, 1, 0, 0, 0, 1])
+  gens = Vector{Hecke.NfRelElem{nf_elem}}[map(E, [1, 0, 0]), map(E, [0, 1, 0]), map(E, [0, 0, 1])]
+  L = hermitian_lattice(E, generators = gens, gram_ambient_space = D)
+
+  @test sprint(show, "text/plain", L) isa String
+  @test get_attribute(L, :absolute_pseudo_matrix) === nothing
+  PMabs = @inferred Hecke.absolute_pseudo_matrix(L)
+  Eabs = nf(base_ring(PMabs))
+  @test matrix(Eabs.(L.pmat.matrix)) == PMabs.matrix
```

Comment:
are you aware of `map_entries`?

Reply:
Yes, I forget it everytime. But here I guess it is just a `change_base_ring` that I wanted to do, I can correct it